shemas/shemaContact.js

```diff
@@ -0,0 +1,12 @@
+
+const Joi = require("joi");
+
+const addShema = Joi.object({
+     name: Joi.string().required(),
+     email: Joi.string().required(),
```

Comment:
You should validate email and phone properly

Reply:
fix validate